elm.html.markdown

```diff
@@ -0,0 +1,267 @@
+---
+language: elm
+contributors:
+    - ["Glen De Cauwsemaecker", "http://glendc.com"]
+filename: learnelm.elm
+---
+
+[Elm](http://elm-lang.org) is a pure functional programming language for writing web-apps in a safe and intuitive way. Elm uses reactivity on top of the functional style, to provide a bridge between the app and the outside world, without giving up the advantages it has from being pure and functional.
+
+You can experiment with the language via [http://elm-lang.org/try](http://elm-lang.org/try). The [syntax guide](http://elm-lang.org/docs/syntax) is a good start point to learn the language, and a lot of this guide is based on that one. If you're already familiar with Javascript, which is the language elm compiles to, you might want to read [http://elm-lang.org/docs/from-javascript](http://elm-lang.org/docs/from-javascript).
+
+We only cover the basics here. After this you might want to learn more about Signals, which allow you to react on user input and other changes from the outside world. You can learn more about that [here](http://elm-lang.org/guide/reactivity#signals).
+
+```elm
```

Comment:
All looks great! Only one problem. elm is not supported in the syntax highlighter library. Look through [this list](https://github.com/adambard/rouge/tree/master/lib/rouge/lexers) of files to see what language is closest to elm and use that one.